api/api.js

```diff
@@ -1,11 +1,11 @@
-var hostname = require('os').hostname(),
-  Router = require('./lib/router.js'),
-  RadarApi = require('./apis/radar.js')
+var hostname = require('os').hostname()
+var Router = require('./lib/router.js')
+var RadarApi = require('./apis/radar.js')
 
 var api = new Router()
 
 function homepage (req, res) {
-  res.setHeader('Content-Type', 'text/plain'); // IE will otherwise try to save the response instead of just showing it.
+  res.setHeader('Content-Type', 'text/plain') // IE will otherwise try to save the response instead of just showing it
```

Comment:
Removed the full stop from the comment. :D why? Is standard an enemy of good English? :D